src/dialog/tooltip/tooltip.directive.ts

```diff
@@ -51,7 +51,11 @@ export class TooltipDirective extends DialogDirective {
 	// tslint:disable-next-line:no-input-rename
 	@Input("tooltip-type") tooltipType: "warning" | "error" | "" = "";
 
-	@HostBinding("attr.aria-describedby") descriptorId: string;
+	@HostBinding("attr.aria-describedby") get descriptorId(): string {
+		if (this.expanded) {
```

Comment:
`else`? Technically, yes, it's going to be `undefined` but we can be more deliberate about it. Looks a bit unfinished this way. (or a `return null` after the `if`)

Whole getter body could be

```typescript
	return this.expanded ? this.dialogConfig.compID : null;
```